frontend/src/Pages/HelpfulLinksManagement.tsx

```diff
@@ -54,11 +54,10 @@ export default function HelpfulLinksManagement() {
         const response = await API.delete(`helpful-links/${id}`);
         if (response.success) {
             updateLinks();
+            toaster('Deleted helpful link successfully', ToastState.success);
```

Comment:
Can you reword this so its similar to Administrators page? "Helpful link deleted successfully"